frontend/api/api.ts

```diff
@@ -45,7 +45,7 @@ export const ImageToText = async (
     labels: JSON.stringify(fieldNames),
   });
 
-  const imageToTextURL = `${middlewareURL}api/image_file_to_text`;
+  const imageToTextURL = `${middlewareURL}image_to_text`;
```

Comment:
Do we want to put dashes `(-)` instead of underscores `(_)` here?  For example the url will come back as `http://localhost:8000/image-to-text` versus `http://localhost:8000/image_to_text`.

Reply:
We can change the URL in the middleware at a later point. For now this should be ok.